CIP-xxxx/README.md

```diff
@@ -0,0 +1,136 @@
+---
+CIP: xxxx
+Title: CBOR Tags for Cardano Data Structures and Registry
+Status: Proposed
+Category: Meta
```

Comment:
```suggestion
Category: Metadata
```
Probably more appropriate?

Reply:
I did the same whoops when I unthinkingly set #751 to `Meta`... but is it certain that all CBOR will be on-chain metadata?  And even so, might it not be parsed off-chain, and if so then perhaps a `Tools` category would be more general & therefore more appropriate?

Reply:
Yes I struggled with the appropriate category for this, because it could be found (one day) in the block/transaction itself on-chain.  In metadata on-chain, and also used off-chain.  Whatever the consensus is where this should live I am happy with it.  `Tools` is probably the broadest category because it is mostly about interoperability.

Reply:
@stevenj @Ryun1 let's plan to decide, with opportunity for input from community implementors, whether this and #753 are `Tools` or `Metadata` no later than the next CIP meeting; I've put both on agenda (where they would have been due for Triage anyway): https://hackmd.io/@cip-editors/81

Reply:
I think either would be fine, happy to defer to a group decision though

Reply:
thanks @stevenj for your flexibility - in fact this week's CIP meeting concluded by undisputed consensus that the CBOR Registry related CIPs and CPS are of `Category:` = `Tools` (rather than `Meta` or `Metadata`) in short because these formats will inevitably be consumed off-chain by a currently unknowable variety of applications.  cc @Ryun1